detections/cloud/high_number_of_login_failures_from_a_single_source.yml

```diff
@@ -2,7 +2,7 @@ name: High Number of Login Failures from a single source
 id: 7f398cfb-918d-41f4-8db8-2e2474e02222
 version: 1
 date: '2020-12-16'
-description: This search will detect more than 5 login failures in Office365 Azure Active Directory from a single source IP address. Please adjust the threshold value of 5 as suited for your environment.
+description: This search will detect more than 5 login failures in Office365 Azure Active Directory from a single source IP address. Please adjust the threshold value of 5 as suited for your environment. Deprecated because duplicate.
```

Comment:
@P4T12ICK : this search is not completely a duplicate. This search explicitly looks for 5 login failures happening from a particular src_ip

Reply:
I believe the other one `O365 Excessive Authentication Failures Alert` is looking for multiple login failures by user.

Reply:
fixed this one